ktlint/src/main/kotlin/com/pinterest/ktlint/internal/KtlintCommandLine.kt

```diff
@@ -217,6 +240,11 @@ internal class KtlintCommandLine {
         }
         logger = configureLogger()
 
+        assertStdinAndPatternsFromStdinOptionsMutuallyExclusive()
+
+        val stdinPatterns: Set<String> = readPatternsFromStdin()
```

Comment:
Please add a CLI Test for the new flag. See SimpleCLITest for example of such a test.